tasks/pruner.yml

```diff
@@ -0,0 +1,41 @@
+- name: Set Facts
+  set_fact:
+      osbs_build_pruner_cronjob_yaml: "{{ osbs_pruner_template_dest }}/build-pruner-cronjob.yaml"
+
+- name: Query pruner service account
```

Comment:
For service account and clusterrolebinding manipulation, can we use the same pattern that is done in `tasks/main.yml`? It's a little more resilient.

Reply:
Yes, and for the CronJob. Updated.